compute_tools/src/compute.rs

```diff
@@ -404,8 +404,8 @@ impl ComputeNode {
     }
 
     /// Select `pg_stat_statements` data and return it as a stringified JSON
-    pub async fn collect_insights(&self) -> String {
```

Comment:
This change doesn't make much sense to me and it's rather misleading. Previously it was returning a complete json, so the idea was to pack more data into it in the future. Now it returns raw lines from `pg_stat_statements`, which is weird and doesn't correspond to its name and description

Anyway, generally -1 for this change

+1 for using `serde_json::json!`, though, so I'd just move it inside `collect_insights` and keep function signature intact

Reply:
Hmm, you're already returning the JSON from the database. My changes will break the response. Will rollback these changes

Reply:
> Hmm, you're already returning the JSON from the database

Yep, I didn't think that `serde_json::json!` will probably do another round of escaping, indeed

Reply:
reverted